requirements.txt

```diff
@@ -1,3 +1,3 @@
-asyncpg==0.18.3
-SQLAlchemy>=1.2
-aiocontextvars==0.2.1
+asyncpg~=0.18
```

Comment:
Can you confirm this is what you want?
https://www.python.org/dev/peps/pep-0440/#compatible-release
0.19 also matches `~=0.18`.

Is it for asyncpg 1.0 release?

Reply:
Yeah, I confirm I wanted `asyncpg>=0.18,<1.0`, but I don't see asyncpg 1.0 coming in the near future, just in case it'll be a breaking release.